packages/devtools_app/lib/src/primitives/url_utils.dart

```diff
@@ -0,0 +1,84 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// TODO(kenz): try to eliminate this method once
+// https://github.com/dart-lang/sdk/issues/46872 is fixed.
+/// Returns a simplified version of a package url, replacing "path/to/flutter"
+/// with "package:".
+///
+/// This is a bit of a hack, as we have file paths instead of the explicit
+/// "package:uris" we'd like to have. This will be problematic for a use case
+/// such as "packages/my_package/src/utils/packages/flutter/".
+String getSimplePackageUrl(String url) {
```

Comment:
can you remove this method and associated tests? (essentially removing everything that was removed as part of https://github.com/flutter/devtools/pull/3932/files#diff-0fb55973e4befafb1d8f5ee26a619a6500f6524ef0dfcf2c252f62aa0644076f)

Reply:
Oops, good catch. I had to abort/re-do the merge and apparently didn't re-remove it the second time 😞